### PR TITLE
chore: upgrade to zwave-js v11

### PR DESCRIPTION
**Pull request by AlCalzone**
This PR is a work in progress. I needed to test something with v11, so I fixed the TS errors in the backend.

### I DID NOT touch the frontend. I expect bugs to be there.

### CODE REVIEW

**Comment by robertsLando** [comment]
TODOs:

- [ ] https://github.com/zwave-js/node-zwave-js/blob/v11-dev/docs/getting-started/migrating-to-v11.md#changed-return-types-of-controllerfirmwareupdateota-and-controllerfirmwareupdateotw-methods
- [ ] https://github.com/zwave-js/node-zwave-js/blob/v11-dev/docs/getting-started/migrating-to-v11.md#changed-nodesetvalue-and-virtualnodesetvalue-to-return-a-setvalueresult
- [ ] https://github.com/zwave-js/node-zwave-js/blob/v11-dev/docs/getting-started/migrating-to-v11.md#the-node-removed-event-callback-now-indicates-why-a-node-was-removed
- [ ] https://github.com/zwave-js/node-zwave-js/blob/v11-dev/docs/getting-started/migrating-to-v11.md#removed-several-deprecated-method-signatures-enums-and-properties

**Comment by AlCalzone** [comment]
v11 is being released now. The setValue related types and methods are available under `@zwave-js/cc/safe`.

**Comment by raman325** [comment]
please bump the server to 1.29.0 as part of this PR

**Comment by robertsLando** [comment]
@AlCalzone Could you give a try to the fw updates methods please?

**Comment by robertsLando** [comment]
@raman325 Done

**Comment by AlCalzone** [comment]
Seems to work:
grafik

I'd reword the information though to make it more user friendly:
> **OK** -> Firmware update successful.
> **RestartPending** -> The device will now restart. This will take approximately 40 seconds.
> Afterwards the device will be re-interviewed. Wait until the interview is done before interacting with the device again.

Some pseudo code to generate the dialog content:
```
if success is true: "Firmware update successful"
	if status is OK_WaitingForActivation:
		"The firmware must be activated manually, likely by pushing a button on the device"
	else if status is OK_RestartPending:
		"The device will now restart."
		if waitTime:
			"This will take approximately ${waitTime} seconds."
	else // status is OK_NoRestart
		if waitTime and not reInterview:
			"Please wait ${waitTime} seconds before interacting with the device again"
else: "Firmware update failed"
	switch status
		Error_Timeout: "There was a timeout during the firmware update"
		Error_Checksum: "Invalid checksum"
		Error_TransmissionFailed: "The transmission failed or was aborted"
		Error_InvalidManufacturerID: "Invalid manufacturer ID"
		Error_InvalidFirmwareID: "Invalid firmware ID"
		Error_InvalidFirmwareTarget: "Invalid firmware target"
		Error_InvalidHeaderInformation: // fall through
		Error_InvalidHeaderFormat: "The firmware header is invalid"
		Error_InsufficientMemory: "The device does not have enough memory to perform the firmware update"
		Error_InvalidHardwareVersion: "Invalid hardware version"

if reInterview:
	if waitTime:
		"Afterwards the device will be re-interviewed."
	else
		"The device will now be re-interviewed."
	"Wait until the interview is done before interacting with the device again."
```
